src/ImageSharp/IImageProcessorApplicator{TPixel}.cs

```diff
@@ -0,0 +1,50 @@
+﻿// <copyright file="IImageProcessorApplicator.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp
+{
+    using System;
+    using ImageSharp.Formats;
+    using ImageSharp.PixelFormats;
+    using ImageSharp.Processing;
+    using SixLabors.Primitives;
+
+    /// <summary>
+    /// An interface to queue up image operations.
+    /// </summary>
+    /// <typeparam name="TPixel">The pixel format</typeparam>
+    public interface IImageProcessorApplicator<TPixel>
```

Comment:
Shouldn't we find a less verbose name for this? How about `IImageProcessingContext<TPixel>`? //`image.Mutate(ctx => ctx.Resize(..) )`